api-reference/10 UI Widgets/DraggableBase/1 Configuration/group.md

```diff
@@ -9,3 +9,12 @@ Allows you to group several widgets, so that users can drag and drop items betwe
 
 ---
 Set this option to identical values for the widgets you want to collect in a single group.
+
+To perform cross-component drag and drop, implement the [onRemove]({basewidgetpath}/Configuration/#onRemove) and/or [onAdd]({basewidgetpath}/Configuration/#onAdd) handlers. These handlers are executed when users drop items to the target component.      
```

Comment:
There's no connection with the information given before. Please rewrite this article in a more cohesive manner.